LINUXCOMPILE.md

```diff
@@ -13,32 +13,22 @@
     make install
 
     # Fedora
-    sudo dnf install gcc gcc-c++ hwloc-devel libmicrohttpd-devel openssl-devel cmake
+    sudo dnf install gcc gcc-c++ hwloc-devel libmicrohttpd-devel libstdc++-static make openssl-devel cmake
     cmake .
     make install
 
     # CentOS
-    sudo yum install centos-release-scl cmake3 hwloc-devel libmicrohttpd-devel openssl-devel
-    sudo yum install devtoolset-4-gcc*
+    sudo yum install centos-release-scl epel-release
+    sudo yum install cmake3 devtoolset-4-gcc* hwloc-devel libmicrohttpd-devel openssl-devel make
     sudo scl enable devtoolset-4 bash
     cmake3 .
     make install
-
-    # Ubuntu 14.04
```

Comment:
Could you please keep the documentation for 14.04. If we will remove the documentation than this should be also done for the other version. But body will search in a shell file for a documentation.

If we remove it it should be done in a separate PR where we somehow linking from the  README.md. Maybe we need to separate the docker scripts for each distribution and version so we can point explicitly to a single script.